pkg/apis/extensions/v1alpha1/types_operatingsystemconfig.go

```diff
@@ -218,3 +218,6 @@ const (
 	// CRINameContainerD is a constant for ContainerD CRI name
 	CRINameContainerD = "containerd"
 )
+
+// ContainerDRuntimeContainersBinFolder is the folder where Container Runtime binaries should be saved for ContainerD usage
+const ContainerDRuntimeContainersBinFolder = "/var/bin/cr"
```

Comment:
```suggestion
const ContainerDRuntimeContainersBinFolder = "/var/bin/containerruntimes"
```

Reply:
Does it make sense to add a field to the `extensions.gardener.cloud/v1alpha1.ContainerRuntime` spec?

```
spec:
  binaryFolder: <path-to-binary-folder>
```

Instead of using this constant here? It'll make it more explicit for the CR extension where to put the binaries.

Reply:
Well, I had this dilemma as well.
I decided to put it here because it is ContainerD configurations (CRI config) not exactly CR extensions but CR Extensions do use it.
I can move it to CR it you prefer.

Reply:
Hm, yes, CR extensions use it, so it's probably safer to provide them the path where to put the binaries via the spec. This way they don't need to rely on a constant. The constant, however, can be kept so that gardener can use it while crafting the `ContainerRuntime` resource. WDYT?

Reply:
Ok, so do you mean that i would extended the Container runtime crd to contain the Path as well?

Reply:
Yes, maybe add a new field `.spec.binaryPath` to the `ContainerRuntime` CRD .

Reply:
Ok i will